### DISCUSSION TITLE
Some Values in getSlickOptionsis() is not working in Serenity 8.x version

### DISCUSSION THREAD

**Discussion by s10730047** [serenity-is/Serenity#7140] [Help] [answered]
this is my xxxGrid.ts Code:

protected getSlickOptions() {
      var opt =  super.getSlickOptions();
      opt.rowHeight = RowScaleSelection.getCurrentRowHeight();
      opt.frozenColumns = 2;
      opt.fullWidthRows = true;
      opt.showFooterRow = false;
      opt.selectedCellCssClass = "row-selected-light";
      return opt;
  }

Only rowHeight and selectedCellCssClass are working; the others are not effective on the grid.

**Comment by VictorTomaili** [comment]
`showFooterRow` is for set `this.view.setSummaryOptions` or any other footer feature
for the `frozenColumns` and `fullWidthRows` you need to set layout engine option as `new FrozenLayout()`

Thats all...

**Reply by s10730047** [reply]
Sorry, I means I don't want FooterRow show on Grid,

Like This Code
`opt.showFooterRow = false;
`

**Reply by VictorTomaili** [reply, marked as answer]
What did you mean about footer row?

This is the footer row (summary mixin), and this exists in pro features. 
![image](https://github.com/serenity-is/Serenity/assets/2023440/f39147dd-f946-4964-888c-098bffe1861f)

You may think the footer row is navigation bar on bottom. But this is not the correct config for. 
use below code to remove pager in your grid

```
protected override usePager() { return false; }
```
